docs/windows.rst

```diff
@@ -5,24 +5,24 @@ Building/hacking/using on MS-Windows
 
 ----
 
-This part of the documentation is written by Frank Fesevur, co-maintainer of universal-ctags and the maintainer of the Windows port of this project. It is still very much work in progress. Things still need to be written down, tested or even investigated. When building for Windows you need to know there are many compilers and many building environments. This is a summary of available options and things that have been tested so far.
+This part of the documentation is written by Frank Fesevur, co-maintainer of Universal-ctags and the maintainer of the Windows port of this project. It is still very much work in progress. Things still need to be written down, tested or even investigated. When building for Windows you need to know there are many compilers and many building environments. This is a summary of available options and things that have been tested so far.
 
 
 Compilers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are many compilers for Windows. Apart from the ones mentioned here others do exist, but not used by me.
+There are many compilers for Windows. Apart from the ones mentioned here others do exist, but are not used by me.
```

Comment:
Last sentence could be "Compilers not mentioned here may work but are not tested."